design/data-structure.md

```diff
@@ -44,7 +44,7 @@ JSON-like data strucutres are used when editing JSON-like documents.
 
 - `Primitive`: represents primitive data like `string`, `number`, `boolean`, `null`, etc.
 - `Object`: represents [object type](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object) of JavaScript. Just like JavaScript, you can use `Object` as [hash table](https://en.wikipedia.org/wiki/Hash_table).
-- `Array`: represents [array type](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array) of JavaScript. You can also use `Array` as [list](https://en.wikipedia.org/wiki/List_(abstract_data_type)).
+- `Array`: represents [array type](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array) of JavaScript. You can also use `Array` as [list](<https://en.wikipedia.org/wiki/List_(abstract_data_type)>).
```

Comment:
What are these `<`, `>` for?

Reply:
For some reasons my prettier added these. Removed them in d8427de4584e3faaeb8bd5531de92dadacf99eba.